### ISSUE TITLE
Edit/command mode visual indication gone

### ISSUE THREAD

**Issue by melvinev** [fredericosantos/jupyterlab-fsx-monokai-theme#7]
Hi @fredericosantos 

Small bug here - when you hit escape to go from command to edit mode in Jupyter, there is no longer a visual cue.  The red square with brackets (which I like!) stays the same color, so it is hard to tell which mode the current cursor is in. 

![image](https://user-images.githubusercontent.com/35536427/99314512-19130300-2816-11eb-8a3d-2fcba4a04a4c.png)

**Comment by melvinev** [comment]
For example edit mode:

![image](https://user-images.githubusercontent.com/35536427/99314637-43fd5700-2816-11eb-938c-2def1294ef84.png)

Then I hit escape to switch to command mode, the text cell grays out to indicate the mode switched:
![image](https://user-images.githubusercontent.com/35536427/99314731-61cabc00-2816-11eb-9ce9-5a93eea57c90.png)

**Comment by fredericosantos** [comment]
You're completely right! I don't know why I started being more confused since I changed the UI so much and you nailed it. I'm currently really out of time to update this due to data science projects. I was trying to mimic the blinking effects of vs code of the cursor caret to great disappointment as it seems the blinking is done at something higher than CSS (I'm completely new to this to be honest, I'm learning as I go).

I will issue make this my top priority! Trying to update it today. I might go back to highlighting the whole cell; I removed it since I felt we didn't need it.

I also want to do something about the bar on the left and turn it into a maximize / minimize button? Maybe that's too much clutter... To be tested! Glad you enjoyed it, I will most likely release a blue version when I'm nearing the final prototype.

**Comment by melvinev** [comment]
Seems fixed! Thank you, great new release! @fredericosantos
